tests/test_exclude.py

```diff
@@ -26,26 +28,39 @@ def test_exclude_recursive_negate(tmp_path):
     assert not (tmp_path / "do_not_copy_me.txt").exists()
 
 
-def test_config_exclude(tmp_path):
-    src, dst = tmp_path / "src", tmp_path / "dst"
-    build_file_tree({src / "copier.yml": "_exclude: ['*.txt']", src / "aaaa.txt": ""})
+def test_config_exclude(tmp_path_factory: pytest.TempPathFactory) -> None:
+    src, dst = map(tmp_path_factory.mktemp, ("src", "dst"))
+    build_file_tree(
+        {
+            src / "copier.yml": "_exclude: ['*.txt']",
+            src / "aaaa.txt": "",
+        }
+    )
```

Comment:
Do you agree with this change (and the other related ones in this file)? It's only stylistic, but I think it's much more readable to use a multiline dict when there is more than file to create.

Reply:
Let's undo it too.

Reply:
Oh well ... Done. :shrug: